src/schema/index.js

```diff
@@ -74,7 +82,7 @@ type Block {
 
 type syncInfo {
   syncBlockNum: Int
-  syncBlockTime: String
+  syncBlockTime: Int
```

Comment:
timestamp like `bettingStartTime` is returned as `String`, for consistency, we should use `string` for syncBlockTime

Reply:
if we do that as a `String`, then we should change all `uint256` vars to `String`. I feel like we should be consistent based on what the type is in the contract. What do you think?

Reply:
BTW, all block times are a UNIX timestamp, so just a number

Reply:
I would like to leave it as `Int` for now as its working with the UI right now. If we do change to `String`, I still suggest we do a refactor for all `uint256` values to use `String`.

This will need to change all blockNums to `String`. And if we do that, we will have to refactor a lot of logic where we would need to convert String -> Int to be able to do comparisons.

After thinking about this, I think we should change all timestamps to `Int` instead. I've calculated the max value of JS Int to UNIX calculator and the limit would be: `Tuesday, June 5, 2255 11:47:34.741 PM` so keeping it as an Int would work.

Reply:
👍 changed